libopae/plugins/xfpga/open.c

```diff
@@ -93,6 +93,11 @@ xfpga_fpgaOpen(fpga_token token, fpga_handle *handle, int flags)
 	// Init workspace table
 	_handle->wsid_root = NULL;
 
+	// Init metric enum
+	_handle->metric_enum_status = false;
+	_handle->dl_handle = NULL;
```

Comment:
The name dl_handle is too generic.  Please rename to reflect its association with metrics.